### DISCUSSION TITLE
Skipping workflows with "skip-checks:true" in the commit message

### DISCUSSION THREAD

**Discussion by simbo** [community/community#55569] [Enterprise] [answered]
### Select Topic Area

Question

### Body

Hi,

I'd like to skip workflows for certain commits.
I am on GitHub Enterprise 3.5.

The [documentation](https://docs.github.com/en/enterprise-server@3.5/actions/managing-workflow-runs/skipping-workflow-runs) says:

> Workflows that would otherwise be triggered using on: push or on: pull_request won't be triggered if you add any of the following strings to the commit message in a push, or the HEAD commit of a pull request:
>
> - `[skip ci]`
> - `[ci skip]`
> - `[no ci]`
> - `[skip actions]`
> - `[actions skip]`
>
> Alternatively, you can end the commit message with two empty lines followed by either:
>
> - `skip-checks:true`
> - `skip-checks: true`

I would prefer the alternative option using `skip-checks: true` but I am struggling to achieve this.

Every time I try to create a commit message with two empty consecutive lines git seems to clean that up and only one empty line is left. Is it even possible to create two empty lines in a commit message?

I tried this:

```sh
git commit -m "whatever


skip-checks: true"
```

```sh
git commit -m $'whatever\n\n\nskip-checks: true'
```

It always generates only one empty line and the workflows start anyway.

Any ideas?

Simon

**Comment by jge162** [comment]
Git automatically trims consecutive empty lines to a single empty line.

**Reply by jge162** [reply, marked as answer]
One option is Check if there are any Git configurations or hooks that modify or alter the commit message. These configurations/hooks could be interfering with the expected behavior. You can use the following command.

`git config --get core.hooksPath`

If a hooks path is returned, navigate to that directory and check for any pre-commit or commit-msg scripts that may be modifying the commit message.

**Comment by simbo** [comment]
I have husky checking on commits and on pushes.
But it doesn't modify the message, it just checks formatting and prefixes.
Disabling husky doesn't also change the described behavior.

I am just wondering why GitHub offers that option with the two empty lines, while probably being aware that it's hard to impossible to create multiple consecutive lines in a commit message. 🤷‍♂️

**Reply by jge162** [reply]
GitHub allows you to include two empty lines in the commit message template, it's important to note that `Git` itself has its own conventions and limitations for commit messages.

**Comment by jeyaramashok** [comment]
is there a way to add two empty lines to the commit message?

my local shows empty lines
```
commit feecd0d8e2358e7f53fd7bd1551bd8ef7fee83dc (HEAD -> e2e-test-3, origin/e2e-test-3)
Author: <xyz>
Date:   Tue Sep 19 09:55:34 2023 -0700

    [ci skip] testing status check


    skip-checks:true
```
but not the remote

![Screenshot 2023-09-19 at 10 04 42 AM](https://github.com/community/community/assets/6378458/0b62912c-2f40-4080-b156-fe1576b37088)
